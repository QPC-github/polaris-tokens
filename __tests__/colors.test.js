/* eslint-env jest */

const fs = require('fs');
const path = require('path');

const colorFiles = [
  'colors.color-map.scss',
  'colors.custom-properties.css',
  'colors.json',
  'colors.scss',
  'Polaris.sketchpalette',
  'index.d.ts',
];

colorFiles.map((filename) =>
  it(`renders ${filename} correctly`, () => {
    const contents = fs.readFileSync(
      path.join(__dirname, '..', 'dist', filename),
      {
        encoding: 'utf-8',
      },
    );
    expect(contents).toMatchSnapshot();
  }),
);
