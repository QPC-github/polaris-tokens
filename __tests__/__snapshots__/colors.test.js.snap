// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Polaris.sketchpalette correctly 1`] = `
"{
  \\"compatibleVersion\\": \\"2.0\\",
  \\"pluginVersion\\": \\"2.0\\",
  \\"colors\\": [
    { \\"red\\": 0, \\"green\\": 0, \\"blue\\": 0, \\"alpha\\": 1 },
    { \\"red\\": 1, \\"green\\": 1, \\"blue\\": 1, \\"alpha\\": 1 },
    {
      \\"red\\": 0.9215686274509803,
      \\"green\\": 0.9607843137254902,
      \\"blue\\": 0.9803921568627451,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7058823529411765,
      \\"green\\": 0.8823529411764706,
      \\"blue\\": 0.9803921568627451,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0,
      \\"green\\": 0.47843137254901963,
      \\"blue\\": 0.807843137254902,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.03137254901960784,
      \\"green\\": 0.3058823529411765,
      \\"blue\\": 0.5411764705882353,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0,
      \\"green\\": 0.0784313725490196,
      \\"blue\\": 0.1607843137254902,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.24313725490196078,
      \\"green\\": 0.3058823529411765,
      \\"blue\\": 0.3411764705882353,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.8901960784313725,
      \\"green\\": 0.9450980392156862,
      \\"blue\\": 0.8745098039215686,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7333333333333333,
      \\"green\\": 0.8980392156862745,
      \\"blue\\": 0.7019607843137254,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.3137254901960784,
      \\"green\\": 0.7215686274509804,
      \\"blue\\": 0.23529411764705882,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.06274509803921569,
      \\"green\\": 0.5019607843137255,
      \\"blue\\": 0.2627450980392157,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.09019607843137255,
      \\"green\\": 0.21176470588235294,
      \\"blue\\": 0.18823529411764706,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.2549019607843137,
      \\"green\\": 0.30980392156862746,
      \\"blue\\": 0.24313725490196078,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9568627450980393,
      \\"green\\": 0.9607843137254902,
      \\"blue\\": 0.9803921568627451,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7019607843137254,
      \\"green\\": 0.7372549019607844,
      \\"blue\\": 0.9607843137254902,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.3607843137254902,
      \\"green\\": 0.41568627450980394,
      \\"blue\\": 0.7686274509803922,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.12549019607843137,
      \\"green\\": 0.1803921568627451,
      \\"blue\\": 0.47058823529411764,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0,
      \\"green\\": 0.023529411764705882,
      \\"blue\\": 0.2235294117647059,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.24313725490196078,
      \\"green\\": 0.2549019607843137,
      \\"blue\\": 0.3333333333333333,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.5686274509803921,
      \\"green\\": 0.6196078431372549,
      \\"blue\\": 0.6705882352941176,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.38823529411764707,
      \\"green\\": 0.45098039215686275,
      \\"blue\\": 0.5058823529411764,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.27058823529411763,
      \\"green\\": 0.30980392156862746,
      \\"blue\\": 0.3568627450980392,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.12941176470588237,
      \\"green\\": 0.16862745098039217,
      \\"blue\\": 0.21176470588235294,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9882352941176471,
      \\"green\\": 0.9215686274509803,
      \\"blue\\": 0.8588235294117647,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 1,
      \\"green\\": 0.7725490196078432,
      \\"blue\\": 0.5450980392156862,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9568627450980393,
      \\"green\\": 0.5764705882352941,
      \\"blue\\": 0.25882352941176473,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7529411764705882,
      \\"green\\": 0.3411764705882353,
      \\"blue\\": 0.09019607843137255,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.2901960784313726,
      \\"green\\": 0.08235294117647059,
      \\"blue\\": 0.01568627450980392,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.34901960784313724,
      \\"green\\": 0.26666666666666666,
      \\"blue\\": 0.18823529411764706,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9647058823529412,
      \\"green\\": 0.9411764705882353,
      \\"blue\\": 0.9921568627450981,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.8901960784313725,
      \\"green\\": 0.8156862745098039,
      \\"blue\\": 1,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.611764705882353,
      \\"green\\": 0.41568627450980394,
      \\"blue\\": 0.8705882352941177,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.3137254901960784,
      \\"green\\": 0.1411764705882353,
      \\"blue\\": 0.5607843137254902,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.13725490196078433,
      \\"green\\": 0,
      \\"blue\\": 0.3176470588235294,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.3137254901960784,
      \\"green\\": 0.28627450980392155,
      \\"blue\\": 0.35294117647058826,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.984313725490196,
      \\"green\\": 0.9176470588235294,
      \\"blue\\": 0.8980392156862745,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.996078431372549,
      \\"green\\": 0.6784313725490196,
      \\"blue\\": 0.6039215686274509,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9294117647058824,
      \\"green\\": 0.38823529411764707,
      \\"blue\\": 0.2784313725490196,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7490196078431373,
      \\"green\\": 0.027450980392156862,
      \\"blue\\": 0.06666666666666667,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.2,
      \\"green\\": 0.00392156862745098,
      \\"blue\\": 0.00392156862745098,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.34509803921568627,
      \\"green\\": 0.23529411764705882,
      \\"blue\\": 0.20784313725490197,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9764705882352941,
      \\"green\\": 0.9803921568627451,
      \\"blue\\": 0.984313725490196,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9568627450980393,
      \\"green\\": 0.9647058823529412,
      \\"blue\\": 0.9725490196078431,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.8745098039215686,
      \\"green\\": 0.8901960784313725,
      \\"blue\\": 0.9098039215686274,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7686274509803922,
      \\"green\\": 0.803921568627451,
      \\"blue\\": 0.8352941176470589,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.8784313725490196,
      \\"green\\": 0.9607843137254902,
      \\"blue\\": 0.9607843137254902,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.7176470588235294,
      \\"green\\": 0.9254901960784314,
      \\"blue\\": 0.9254901960784314,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.2784313725490196,
      \\"green\\": 0.7568627450980392,
      \\"blue\\": 0.7490196078431373,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0,
      \\"green\\": 0.5176470588235295,
      \\"blue\\": 0.5568627450980392,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0,
      \\"green\\": 0.19215686274509805,
      \\"blue\\": 0.20784313725490197,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.25098039215686274,
      \\"green\\": 0.3254901960784314,
      \\"blue\\": 0.3215686274509804,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9882352941176471,
      \\"green\\": 0.9450980392156862,
      \\"blue\\": 0.803921568627451,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 1,
      \\"green\\": 0.9176470588235294,
      \\"blue\\": 0.5411764705882353,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.9333333333333333,
      \\"green\\": 0.7607843137254902,
      \\"blue\\": 0,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.611764705882353,
      \\"green\\": 0.43529411764705883,
      \\"blue\\": 0.09803921568627451,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.3411764705882353,
      \\"green\\": 0.23137254901960785,
      \\"blue\\": 0,
      \\"alpha\\": 1
    },
    {
      \\"red\\": 0.34901960784313724,
      \\"green\\": 0.3176470588235294,
      \\"blue\\": 0.18823529411764706,
      \\"alpha\\": 1
    }
  ],
  \\"gradients\\": [],
  \\"images\\": []
}
"
`;

exports[`renders colors.color-map.scss correctly 1`] = `
"$colors: (
  black: (base: rgb(0, 0, 0)),
  white: (base: rgb(255, 255, 255)),
  blue:
    (
      lighter: rgb(235, 245, 250),
      light: rgb(180, 225, 250),
      base: rgb(0, 122, 206),
      dark: rgb(8, 78, 138),
      darker: rgb(0, 20, 41),
      text: rgb(62, 78, 87)
    ),
  green:
    (
      lighter: rgb(227, 241, 223),
      light: rgb(187, 229, 179),
      base: rgb(80, 184, 60),
      dark: rgb(16, 128, 67),
      darker: rgb(23, 54, 48),
      text: rgb(65, 79, 62)
    ),
  indigo:
    (
      lighter: rgb(244, 245, 250),
      light: rgb(179, 188, 245),
      base: rgb(92, 106, 196),
      dark: rgb(32, 46, 120),
      darker: rgb(0, 6, 57),
      text: rgb(62, 65, 85)
    ),
  ink:
    (
      lightest: rgb(145, 158, 171),
      lighter: rgb(99, 115, 129),
      light: rgb(69, 79, 91),
      base: rgb(33, 43, 54)
    ),
  orange:
    (
      lighter: rgb(252, 235, 219),
      light: rgb(255, 197, 139),
      base: rgb(244, 147, 66),
      dark: rgb(192, 87, 23),
      darker: rgb(74, 21, 4),
      text: rgb(89, 68, 48)
    ),
  purple:
    (
      lighter: rgb(246, 240, 253),
      light: rgb(227, 208, 255),
      base: rgb(156, 106, 222),
      dark: rgb(80, 36, 143),
      darker: rgb(35, 0, 81),
      text: rgb(80, 73, 90)
    ),
  red:
    (
      lighter: rgb(251, 234, 229),
      light: rgb(254, 173, 154),
      base: rgb(237, 99, 71),
      dark: rgb(191, 7, 17),
      darker: rgb(51, 1, 1),
      text: rgb(88, 60, 53)
    ),
  sky:
    (
      lighter: rgb(249, 250, 251),
      light: rgb(244, 246, 248),
      base: rgb(223, 227, 232),
      dark: rgb(196, 205, 213)
    ),
  teal:
    (
      lighter: rgb(224, 245, 245),
      light: rgb(183, 236, 236),
      base: rgb(71, 193, 191),
      dark: rgb(0, 132, 142),
      darker: rgb(0, 49, 53),
      text: rgb(64, 83, 82)
    ),
  yellow:
    (
      lighter: rgb(252, 241, 205),
      light: rgb(255, 234, 138),
      base: rgb(238, 194, 0),
      dark: rgb(156, 111, 25),
      darker: rgb(87, 59, 0),
      text: rgb(89, 81, 48)
    )
);
"
`;

exports[`renders colors.custom-properties.css correctly 1`] = `
":root {
  --color-black-base: rgb(0, 0, 0);
  --color-white-base: rgb(255, 255, 255);
  --color-blue-lighter: rgb(235, 245, 250);
  --color-blue-light: rgb(180, 225, 250);
  /* This is a description and it’s awesome. LOL */
  --color-blue-base: rgb(0, 122, 206);
  /* This is some documentation */
  --color-blue-dark: rgb(8, 78, 138);
  /* This blue is great for things */
  --color-blue-darker: rgb(0, 20, 41);
  --color-blue-text: rgb(62, 78, 87);
  --color-green-lighter: rgb(227, 241, 223);
  --color-green-light: rgb(187, 229, 179);
  --color-green-base: rgb(80, 184, 60);
  --color-green-dark: rgb(16, 128, 67);
  --color-green-darker: rgb(23, 54, 48);
  --color-green-text: rgb(65, 79, 62);
  --color-indigo-lighter: rgb(244, 245, 250);
  --color-indigo-light: rgb(179, 188, 245);
  --color-indigo-base: rgb(92, 106, 196);
  --color-indigo-dark: rgb(32, 46, 120);
  --color-indigo-darker: rgb(0, 6, 57);
  --color-indigo-text: rgb(62, 65, 85);
  --color-ink-lightest: rgb(145, 158, 171);
  --color-ink-lighter: rgb(99, 115, 129);
  --color-ink-light: rgb(69, 79, 91);
  --color-ink-base: rgb(33, 43, 54);
  --color-orange-lighter: rgb(252, 235, 219);
  --color-orange-light: rgb(255, 197, 139);
  --color-orange-base: rgb(244, 147, 66);
  --color-orange-dark: rgb(192, 87, 23);
  --color-orange-darker: rgb(74, 21, 4);
  --color-orange-text: rgb(89, 68, 48);
  --color-purple-lighter: rgb(246, 240, 253);
  --color-purple-light: rgb(227, 208, 255);
  --color-purple-base: rgb(156, 106, 222);
  --color-purple-dark: rgb(80, 36, 143);
  --color-purple-darker: rgb(35, 0, 81);
  --color-purple-text: rgb(80, 73, 90);
  --color-red-lighter: rgb(251, 234, 229);
  --color-red-light: rgb(254, 173, 154);
  --color-red-base: rgb(237, 99, 71);
  --color-red-dark: rgb(191, 7, 17);
  --color-red-darker: rgb(51, 1, 1);
  --color-red-text: rgb(88, 60, 53);
  --color-sky-lighter: rgb(249, 250, 251);
  --color-sky-light: rgb(244, 246, 248);
  --color-sky-base: rgb(223, 227, 232);
  --color-sky-dark: rgb(196, 205, 213);
  --color-teal-lighter: rgb(224, 245, 245);
  --color-teal-light: rgb(183, 236, 236);
  --color-teal-base: rgb(71, 193, 191);
  --color-teal-dark: rgb(0, 132, 142);
  --color-teal-darker: rgb(0, 49, 53);
  --color-teal-text: rgb(64, 83, 82);
  --color-yellow-lighter: rgb(252, 241, 205);
  --color-yellow-light: rgb(255, 234, 138);
  --color-yellow-base: rgb(238, 194, 0);
  --color-yellow-dark: rgb(156, 111, 25);
  --color-yellow-darker: rgb(87, 59, 0);
  --color-yellow-text: rgb(89, 81, 48);
}
"
`;

exports[`renders colors.json correctly 1`] = `
"{
  \\"color-black-base\\": \\"rgb(0, 0, 0)\\",
  \\"color-white-base\\": \\"rgb(255, 255, 255)\\",
  \\"color-blue-lighter\\": \\"rgb(235, 245, 250)\\",
  \\"color-blue-light\\": \\"rgb(180, 225, 250)\\",
  \\"color-blue-base\\": \\"rgb(0, 122, 206)\\",
  \\"color-blue-dark\\": \\"rgb(8, 78, 138)\\",
  \\"color-blue-darker\\": \\"rgb(0, 20, 41)\\",
  \\"color-blue-text\\": \\"rgb(62, 78, 87)\\",
  \\"color-green-lighter\\": \\"rgb(227, 241, 223)\\",
  \\"color-green-light\\": \\"rgb(187, 229, 179)\\",
  \\"color-green-base\\": \\"rgb(80, 184, 60)\\",
  \\"color-green-dark\\": \\"rgb(16, 128, 67)\\",
  \\"color-green-darker\\": \\"rgb(23, 54, 48)\\",
  \\"color-green-text\\": \\"rgb(65, 79, 62)\\",
  \\"color-indigo-lighter\\": \\"rgb(244, 245, 250)\\",
  \\"color-indigo-light\\": \\"rgb(179, 188, 245)\\",
  \\"color-indigo-base\\": \\"rgb(92, 106, 196)\\",
  \\"color-indigo-dark\\": \\"rgb(32, 46, 120)\\",
  \\"color-indigo-darker\\": \\"rgb(0, 6, 57)\\",
  \\"color-indigo-text\\": \\"rgb(62, 65, 85)\\",
  \\"color-ink-lightest\\": \\"rgb(145, 158, 171)\\",
  \\"color-ink-lighter\\": \\"rgb(99, 115, 129)\\",
  \\"color-ink-light\\": \\"rgb(69, 79, 91)\\",
  \\"color-ink-base\\": \\"rgb(33, 43, 54)\\",
  \\"color-orange-lighter\\": \\"rgb(252, 235, 219)\\",
  \\"color-orange-light\\": \\"rgb(255, 197, 139)\\",
  \\"color-orange-base\\": \\"rgb(244, 147, 66)\\",
  \\"color-orange-dark\\": \\"rgb(192, 87, 23)\\",
  \\"color-orange-darker\\": \\"rgb(74, 21, 4)\\",
  \\"color-orange-text\\": \\"rgb(89, 68, 48)\\",
  \\"color-purple-lighter\\": \\"rgb(246, 240, 253)\\",
  \\"color-purple-light\\": \\"rgb(227, 208, 255)\\",
  \\"color-purple-base\\": \\"rgb(156, 106, 222)\\",
  \\"color-purple-dark\\": \\"rgb(80, 36, 143)\\",
  \\"color-purple-darker\\": \\"rgb(35, 0, 81)\\",
  \\"color-purple-text\\": \\"rgb(80, 73, 90)\\",
  \\"color-red-lighter\\": \\"rgb(251, 234, 229)\\",
  \\"color-red-light\\": \\"rgb(254, 173, 154)\\",
  \\"color-red-base\\": \\"rgb(237, 99, 71)\\",
  \\"color-red-dark\\": \\"rgb(191, 7, 17)\\",
  \\"color-red-darker\\": \\"rgb(51, 1, 1)\\",
  \\"color-red-text\\": \\"rgb(88, 60, 53)\\",
  \\"color-sky-lighter\\": \\"rgb(249, 250, 251)\\",
  \\"color-sky-light\\": \\"rgb(244, 246, 248)\\",
  \\"color-sky-base\\": \\"rgb(223, 227, 232)\\",
  \\"color-sky-dark\\": \\"rgb(196, 205, 213)\\",
  \\"color-teal-lighter\\": \\"rgb(224, 245, 245)\\",
  \\"color-teal-light\\": \\"rgb(183, 236, 236)\\",
  \\"color-teal-base\\": \\"rgb(71, 193, 191)\\",
  \\"color-teal-dark\\": \\"rgb(0, 132, 142)\\",
  \\"color-teal-darker\\": \\"rgb(0, 49, 53)\\",
  \\"color-teal-text\\": \\"rgb(64, 83, 82)\\",
  \\"color-yellow-lighter\\": \\"rgb(252, 241, 205)\\",
  \\"color-yellow-light\\": \\"rgb(255, 234, 138)\\",
  \\"color-yellow-base\\": \\"rgb(238, 194, 0)\\",
  \\"color-yellow-dark\\": \\"rgb(156, 111, 25)\\",
  \\"color-yellow-darker\\": \\"rgb(87, 59, 0)\\",
  \\"color-yellow-text\\": \\"rgb(89, 81, 48)\\"
}
"
`;

exports[`renders colors.scss correctly 1`] = `
"$color-black-base: rgb(0, 0, 0);
$color-white-base: rgb(255, 255, 255);
$color-blue-lighter: rgb(235, 245, 250);
$color-blue-light: rgb(180, 225, 250);
// This is a description and it’s awesome. LOL
$color-blue-base: rgb(0, 122, 206);
// This is some documentation
$color-blue-dark: rgb(8, 78, 138);
// This blue is great for things
$color-blue-darker: rgb(0, 20, 41);
$color-blue-text: rgb(62, 78, 87);
$color-green-lighter: rgb(227, 241, 223);
$color-green-light: rgb(187, 229, 179);
$color-green-base: rgb(80, 184, 60);
$color-green-dark: rgb(16, 128, 67);
$color-green-darker: rgb(23, 54, 48);
$color-green-text: rgb(65, 79, 62);
$color-indigo-lighter: rgb(244, 245, 250);
$color-indigo-light: rgb(179, 188, 245);
$color-indigo-base: rgb(92, 106, 196);
$color-indigo-dark: rgb(32, 46, 120);
$color-indigo-darker: rgb(0, 6, 57);
$color-indigo-text: rgb(62, 65, 85);
$color-ink-lightest: rgb(145, 158, 171);
$color-ink-lighter: rgb(99, 115, 129);
$color-ink-light: rgb(69, 79, 91);
$color-ink-base: rgb(33, 43, 54);
$color-orange-lighter: rgb(252, 235, 219);
$color-orange-light: rgb(255, 197, 139);
$color-orange-base: rgb(244, 147, 66);
$color-orange-dark: rgb(192, 87, 23);
$color-orange-darker: rgb(74, 21, 4);
$color-orange-text: rgb(89, 68, 48);
$color-purple-lighter: rgb(246, 240, 253);
$color-purple-light: rgb(227, 208, 255);
$color-purple-base: rgb(156, 106, 222);
$color-purple-dark: rgb(80, 36, 143);
$color-purple-darker: rgb(35, 0, 81);
$color-purple-text: rgb(80, 73, 90);
$color-red-lighter: rgb(251, 234, 229);
$color-red-light: rgb(254, 173, 154);
$color-red-base: rgb(237, 99, 71);
$color-red-dark: rgb(191, 7, 17);
$color-red-darker: rgb(51, 1, 1);
$color-red-text: rgb(88, 60, 53);
$color-sky-lighter: rgb(249, 250, 251);
$color-sky-light: rgb(244, 246, 248);
$color-sky-base: rgb(223, 227, 232);
$color-sky-dark: rgb(196, 205, 213);
$color-teal-lighter: rgb(224, 245, 245);
$color-teal-light: rgb(183, 236, 236);
$color-teal-base: rgb(71, 193, 191);
$color-teal-dark: rgb(0, 132, 142);
$color-teal-darker: rgb(0, 49, 53);
$color-teal-text: rgb(64, 83, 82);
$color-yellow-lighter: rgb(252, 241, 205);
$color-yellow-light: rgb(255, 234, 138);
$color-yellow-base: rgb(238, 194, 0);
$color-yellow-dark: rgb(156, 111, 25);
$color-yellow-darker: rgb(87, 59, 0);
$color-yellow-text: rgb(89, 81, 48);
"
`;
